frontend/templates/views/partials/structured-data.j2

```diff
@@ -60,7 +60,7 @@
 {% set title_prefix = _('Success Story: ') %}
 {% endif %}
 
-<title>{{ title_prefix }}{{ title }} - amp.dev</title>
+<title>{{title_prefix|e}}{{title|e}} - amp.dev</title>
```

Comment:
```suggestion
<title>{{title_prefix}}{{title}} - amp.dev</title>
```

What's inside the `<title>` tag shouldn't be escaped. As I mentioned I think they wouldn't be unescaped when shown in the browser title bar but are gracefully handled by Google for example. That's why we initially used `<` over `&lt;`.